src/controllers/rooms.js

```diff
@@ -0,0 +1,47 @@
+import { Accomodation, Room } from '../database/models';
+
+export default {
+  makeRooms: async (req, res) => {
+    const { id } = req.params;
+    const { roomType } = req.body;
+    const accomodation = await Accomodation.findByPk(id);
+    if (!accomodation) {
+      return res
+        .status(404)
+        .send({ message: res.__('No accommodation found') });
+    }
```

Comment:
I would recommend t use `res.json` instead of `res.send`

Reply:
Thanks done